src/main/kotlin/no/nav/syfo/client/pdl/domain/PdlError.kt

```diff
@@ -5,7 +5,9 @@ data class PdlError(
     val locations: List<PdlErrorLocation>,
     val path: List<String>?,
     val extensions: PdlErrorExtension,
-)
+) {
+    fun is404() = this.extensions.code == "not_found"
+}
```

Comment:
Er det egentlig en 404? Vet ikke om det finnes tilfeller der vi skulle gjerne sett at `not_found` ga en error i stedet for warning, heller 🤷🏼‍♂️

Reply:
Ja, det er vel ikke nødvendigvis http 404 vi får her. Kanskje kalle den isNotFound() i stedet?

Reply:
Dette er jo graphql, da er alt 200 🆗 🚀 Geir sitt forslag var fint! 😄

Reply:
Ja, kan endre det! Dette er jo GraphQL så blir vel mer riktig å bare skrive `isNotFound`

Reply:
Haha så ikke kommentaren din JML ©️ , glemte å oppdatere siden!